README.md

```diff
@@ -3,7 +3,7 @@ A 10+ hours dataset for Chinese speech recognition
 
 ![WenetSpeech](res/wenetspeech.jpg)
 
-
+# Introduction
```

Comment:
there is no introduction now. please add

Reply:
there is no introduction now. please add